File: src/main/resources/static/management/js/chatroom.js
app.controller("chatRoomCtrl", function ($scope) {

    $scope.chatrooms = [{
        no: 0,
        type: 0,
        title: "방제목",
        description: "대화하고 노실분들 입장하세용",
        category:["태그1","태그2"],
        img: "/chat/img/thumbnail_gif_example.gif",
        max: 100,
        createDate: "2020.07.13",
        private: true,
        manager_id: 1,
        participants: [],
        like: [],
        messages:[],
        accessKey: "1234",
        storageImage:[]
    },{
        no: 1,
        type: 0,
        title: "방제목2",
        description: "대화하고 노실분들 입장하세용2",
        category:["태그1","태그2"],
        img: "/chat/img/thumbnail_gif_example.gif",
        max: 100,
        createDate: "2020.07.13",
        private: true,
        manager_id: 2,
        participants: [],
        like: [],
        messages:[],
        accessKey: "1234",
        storageImage:[]
    }];

    angular.element(document).ready( function () {
        $('#account-table').dataTable();
    });


});

File: src/main/java/com/endspring/pictalk/module/chat/model/ChatRoom.java
package com.endspring.pictalk.module.chat.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.time.LocalDateTime;

@Data
@Document(value="chatroom")
@NoArgsConstructor
@AllArgsConstructor
public class ChatRoom {
    @Id
    private String id;
    private String title;
    private String description;
    private String img;
    private String managerId;
    private String accessKey;
    private String notification;

    private int type;
    private int max;

    private boolean isPrivate;

    private LocalDateTime createDate;

    private Message[] messages;

    private String[] categories;

    private Participant[] participants;

    private Participant[] likes;

    private Image[] savedImages;

    private ChatRoomConfig chatRoomConfig;

    public ChatRoom(String id){
        this.id = id;
    }
}
